addon/search/jump-to-line.js

```diff
@@ -42,10 +36,9 @@
 
       var match;
       if (match = /^\s*([\+\-]?\d+)\s*\:\s*(\d+)\s*$/.exec(posStr)) {
-        cm.setCursor(Pos(interpretLine(cm, match[1]), Number(match[2])))
+        cm.setCursor(interpretLine(cm, match[1]), Number(match[2]))
```

Comment:
Also normally across the codebase semicolons are standard. Would it be reasonable to have the same style here?